src/esm_runscripts/batch_system.py

```diff
@@ -220,6 +250,7 @@ def calculate_requirements(config, cluster=None):
                         )
 
                 else:
+                    # FIXME(PG): ...what? Just continue???
```

Comment:
yes, think about `oasis` or `recom` for fesom2: they are components for ESM-Tools, but they are libraries, meaning they don't need cores themselves.

Reply:
Alright. Then I think we should write something to that effect:

```suggestion
                    # NOTE: For components like oasis or recom, no "own" cores are needed
```